main.cpp

```diff
@@ -1,7 +1,29 @@
-#include "threadSafeQueue.h"
+#include "threadPool.h"
+#include <iostream>
 
+void foo1() {
```

Comment:
remove these tests.
After this PR is merged, your next task is to use gTest to test your thread pool